indico/modules/vc/blueprint.py

```diff
@@ -38,6 +39,8 @@
                  'manage_vc_rooms_modify', RHVCManageEventModify, methods=('GET', 'POST'))
 _bp.add_url_rule('/event/<confId>/manage/videoconference/<service>/<int:event_vc_room_id>/remove',
                  'manage_vc_rooms_remove', RHVCManageEventRemove, methods=('POST',))
+_bp.add_url_rule('/event/<confId>/manage/videoconference/<service>/<int:event_vc_room_id>/remove_all',
```

Comment:
we use dashes instead of underscores in URLs, but maybe a querystring argument makes more sense for this instead of a new endpoint?